app/config.py

```diff
@@ -53,3 +53,5 @@
 
 DEFAULT_FUND_ID = "funding-service-design"
 DEFAULT_ROUND_ID = "summer"
+
+APPLICATION_STATUSES = ("NOT_STARTED", "IN_PROGRESS", "COMPLETED", "SUBMITTED")
```

Comment:
Could we use an Enum rather than a tuple for this? This should make the usages much more readable. i.e. https://docs.python.org/3/library/enum.html

Reply:
Yes that's a great shout - I was a bit worried about the calls not being clear enough 💯